uefi/flash.go

```diff
@@ -142,8 +142,8 @@ func NewFlashImage(buf []byte) (*FlashImage, error) {
 	flash.Master = *master
 
 	// BIOS region
-	biosBase := uint32(uint32(flash.Region.BiosBase) * 0x1000)
-	biosSize := uint32(computeRegionSize(flash.Region.BiosBase, flash.Region.BiosLimit))
+	biosBase := uint32(uint32(flash.Region.BIOS.Base) * 0x1000)
```

Comment:
why the double cast? was it really needed?

Reply:
removed and removed the one below